engine/gamesys/src/gamesys/scripts/script_image.cpp

```diff
@@ -70,7 +72,15 @@ namespace dmScript
     *
     * @name image.load
     * @param buffer [type:string] image data buffer
-    * @param [premult] [type:boolean] optional flag if alpha should be premultiplied. Defaults to `false`
+    * @param options [type:table] A table containing options about how to load the image. Supported entries:
+    *
+    * `premultiply_alpha`
+    * : [type:boolean] if true, premultiply the alpha channel
+    *
+    * `buffer`
+    * : [type:boolean] if true, the 'buffer' parameter of the returned table will be a buffer object (see [ref:buffer.create] for more info)
+    * that can be used for creating texture or atlas resources.
+    *
```

Comment:
What should we call this parameter?

Reply:
I think `buffer` is fine. Do we want more formats in the future? If that is the case then perhaps `format` is better, with values such as `bytes` (ie string) and `buffer`. But perhaps `format` implies image format more than return value type? Perhaps `type` instead?